examples/minimal/node_env.js

```diff
@@ -1,6 +1,6 @@
 require('dotenv-extended').load({
   silent: process.env.SUPPRESS_ENV_ERRORS,
-  errorOnMissing: !process.env.SUPPRESS_ENV_ERRORS,
+  errorOnMissing: process.env.SUPPRESS_ENV_ERRORS,
   errorOnRegex: !process.env.SUPPRESS_ENV_ERRORS,
```

Comment:
sure errorOnMissing should be anebled when the suppress env flag is sent? That seems wrong